crates/nu-command/src/debug/info.rs

```diff
@@ -1,16 +1,29 @@
 use nu_protocol::ast::Call;
 use nu_protocol::engine::{Command, EngineState, Stack};
 use nu_protocol::{
-    record, Category, Example, IntoPipelineData, PipelineData, Record, ShellError, Signature, Span,
-    Type, Value,
+    record, Category, Example, IntoPipelineData, LazyRecord, PipelineData, Record, ShellError,
+    Signature, Span, Type, Value,
 };
 use sysinfo::{Pid, PidExt, ProcessExt, ProcessRefreshKind, RefreshKind, System, SystemExt};
-// Character used to separate directories in a Path Environment variable on windows is ";"
-#[cfg(target_family = "windows")]
-const ENV_PATH_SEPARATOR_CHAR: char = ';';
-// Character used to separate directories in a Path Environment variable on linux/mac/unix is ":"
-#[cfg(not(target_family = "windows"))]
-const ENV_PATH_SEPARATOR_CHAR: char = ':';
+const ENV_PATH_SEPARATOR_CHAR: char = {
+    #[cfg(target_family = "windows")]
+    {
+        ';'
+    }
+    #[cfg(not(target_family = "windows"))]
+    {
+        ':'
+    }
+};
+
+macro_rules! equal_to_any {
```

Comment:
Would rather not have macros if possible.

Reply:
Too bad, that was a pretty clean way of checking different variables for different configs !